pycbc/psd/__init__.py

```diff
@@ -159,35 +164,38 @@ def insert_psd_option_group(parser, output=True):
                           "The options --psd-model, --psd-file, --asd-file, "
                           "and --psd-estimation are mutually exclusive.")
     psd_options.add_argument("--psd-model",
-                          help="Get PSD from given analytical model. ", 
-                          choices=get_psd_model_list())
+                             help="Get PSD from given analytical model. ",
+                             choices=get_psd_model_list())
     psd_options.add_argument("--psd-file",
-                          help="Get PSD using given PSD ASCII file")
+                             help="Get PSD using given PSD ASCII file")
     psd_options.add_argument("--asd-file",
-                          help="Get PSD using given ASD ASCII file")
+                             help="Get PSD using given ASD ASCII file")
     psd_options.add_argument("--psd-estimation",
-                          help="Measure PSD from the data, using given "
-                          "average method.",
-                          choices=["mean", "median", "median-mean"])
-    psd_options.add_argument("--psd-segment-length", type=float, 
-                          help="(Required for --psd-estimation) The segment "
-                               "length for PSD estimation (s)")
-    psd_options.add_argument("--psd-segment-stride", type=float, 
-                          help="(Required for --psd-estimation) The separation"
-                               " between consecutive segments (s)")
-    psd_options.add_argument("--psd-num-segments", type=int, default=None,
-                          help="(Optional, used only with --psd-estimation). "
-                               "If given PSDs will be estimated using only "
-                               "this number of segments. If more data is "
-                               "given than needed to make this number of "
-                               "segments than excess data will not be used in "
-                               "the PSD estimate. If not enough data is given "
-                               "the code will fail.")
-    psd_options.add_argument("--psd-inverse-length", type=float, 
-                          help="(Optional) The maximum length of the impulse"
-                          " response of the overwhitening filter (s)")
+                             help="Measure PSD from the data, using given "
+                             "average method.",
+                             choices=["mean", "median", "median-mean"])
+    psd_options.add_argument("--psd-segment-length", type=float,
+                             help="(Required for --psd-estimation) The "
+                             "segment length for PSD estimation (s)")
+    psd_options.add_argument("--psd-segment-stride", type=float,
+                             help="(Required for --psd-estimation) The "
+                             "separation between consecutive segments (s)")
+    if include_data_options :
```

Comment:
`psd-estimation`, `psd-segment-length`, and `psd-segment-stride` should all be within the `if include_data_options` block.